agent/06_agent_create_cluster.sh

```diff
@@ -163,13 +163,74 @@ function mce_complete_deployment() {
   mce_apply_postinstallation_manifests ${mceManifests}
 }
 
-create_image
-if [[ "${AGENT_DISABLE_AUTOMATED:-}" == "true" ]]; then
-  disable_automated_installation
-fi
+# Generate the PXE artifacts
+function create_pxe_files() {
+    local asset_dir=${1}
+    local openshift_install=${2}
+    "${openshift_install}" --dir="${asset_dir}" --log-level=debug agent create pxe-files
+}
+
+# Setup the environment to allow iPXE booting, by reusing libvirt native features
+# to configure dnsmaq tftp server and pxe boot file. Note that this is supported
+# only for IPv4.
+function setup_pxe_server() {
+    local PXE_SERVER_DIR=${WORKING_DIR}/pxe
+    mkdir -p ${PXE_SERVER_DIR}
+
+    # Create the iPXE boot file
+    local BASEURL=tftp://${PROVISIONING_HOST_EXTERNAL_IP}
+    sudo cat > ${PXE_SERVER_DIR}/boot.ipxe << EOF
+#!ipxe
+kernel ${BASEURL}/agent-vmlinuz.x86_64 initrd=main coreos.live.rootfs_url=${BASEURL}/agent-rootfs.x86_64.img ignition.firstboot ignition.platform.id=metal
```

Comment:
When using the `coreos-installer iso extract pxe` command to extract the pxe artifacts from `agent.x86_64.iso`, it produces files named `agent.x86_64-vmlinuz`, `agent.x86_64-rootfs.img`, `agent.x86_64-initrd.img`.
@pawanpinjarkar could we maybe rename the files output from the `agent create pxe-files` command to match? That will make the transition easier for users who are starting with the coreos-installer workaround on 4.12 & 4.13.

Reply:
FIxed via https://github.com/openshift/installer/pull/7185/commits/ba82593d8cba14c6877b7686d446cf0467065935